cmd/git-sweep/main.go

```
@@ -53,23 +54,76 @@ func printDryRunActions(allAnalyzedBranches []types.AnalyzedBranch) {
-		if branch.Remote != "" {
-			fmt.Printf("  - Delete remote '%s/%s'\n", branch.Remote, branch.Name)
+fmt.Println("\nRemote Deletions:")
+hasRemote := false
+for _, branch := range displayableBranches {
+	// Only print actions for selectable branches with remotes
+	if !(branch.Category == types.CategoryActive || branch.Category == types.CategoryMergedOld || branch.Category == types.CategoryUnmergedOld) {
+		continue
+	}
+	if branch.Remote != "" {
+		fmt.Printf("  - Delete remote '%s/%s'\n", branch.Remote, branch.Name)
+		hasRemote = true
```

golangci-lint (1.64.8)

ineffectual assignment to hasRemote

(ineffassign)